src/diffusers/pipelines/flux/pipeline_flux_controlnet.py

```diff
@@ -856,12 +880,19 @@ def __call__(
                 guidance = torch.tensor([guidance_scale], device=device) if use_guidance else None
                 guidance = guidance.expand(latents.shape[0]) if guidance is not None else None
 
+                if isinstance(controlnet_keep[i], list):
+                    current_controlnet_conditioning_scale = [
+                        c * s for c, s in zip(controlnet_conditioning_scale, controlnet_keep[i])
+                    ]
+                else:
+                    current_controlnet_conditioning_scale = controlnet_conditioning_scale * controlnet_keep[i]
```

Comment:
we should probably stay consistent with the other implementations, is there a reason why you changed this part to be different from the SD3 and SDXL ones?

Reply:
yeah its a mistake, controlnet_conditioning_scale could be a list as well, will make changes